recipes/zoltan/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: zoltan
+    version: 3.83
+
+source:
+    fn: zoltan_distrib_v3.83.tar.gz
+    url: http://www.cs.sandia.gov/~kddevin/Zoltan_Distributions/zoltan_distrib_v3.83.tar.gz
+    sha256: d0d78fdeab7a385c87d3666b8a8dc748994ff04d3fd846872a4845e12d79c1bb
+
+build:
+    skip: True  # [win]
+    number: 0
+    string: 0
+
+requirements:
+    build:
+        - libgfortran 3.0.0
+        - mpich 3.2
+    run:
+        - mpich 3.2*
+test:
+  imports:
+    - zoltan
+
+about:
+    home: http://www.cs.sandia.gov/Zoltan/
+    license: BSD
+    license_file: LICENSE.txt
```

Comment:
Also, `license_family: BSD` should be added here

Reply:
@scopatz  Okay, got it. 
Also, I needed a little advide from you.. Actually I am building a conda-forge package(apart from this one) and I had a doubt. My package(say A) has an optional build time dependency(say B), in a way that suppose if the user has already B installed on his machine then A will install additional features accordingly but if B is not present then A is installed just at a basic level without any problem. So is there some way I can do this with conda-forge without creating two separate packages?
Thanks

Reply:
Well you would end up with two separate packages, but you could use outputs to have these both in one feedstock/recipe: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#outputs-section